frontend/app/pages/results.vue

```diff
@@ -82,19 +90,37 @@ export default {
     }
   },
   watch: {
-    zipCode: async function(zipCode) {
-      try {
-        if (this.zipCodeRegex.test(zipCode) || zipCode === '') {
-          const results = await this.$axios.$get(`${process.env.API_URL}/report/result/${this.zipCodeRegex.test(zipCode) ? `?new_format&zip_code=${zipCode}`: ''}`)
-          this.results = this.normalizeResponse(results)
-          this.updateMap()
-        }
-      } catch (e) {
-        console.log(e)
+    zipCode: async function() {
+      if (this.zipCodeRegex.test(this.zipCode) || this.zipCode === '') {
+        this.getFilterResult()
       }
+    },
+    county: async function() {
+      if (this.county.length > 0 || this.county === '') {
+        this.getFilterResult()
+      }
+    },
+    date: async function() {
+      this.getFilterResult()
     }
   },
   methods: {
+    notAfterToday(date) {
+      return date > this.today
+    },
+    async getFilterResult() {
+      const dateParam = `${this.date !== null ? `&date=${this.date}`: ''}`
+      const countyParam = `${this.county !== null ? `&county=${this.county}`: ''}`
+      const zipCodeParam = `${this.zipCodeRegex.test(this.zipCode) ? `&zip_code=${this.zipCode}`: ''}`
+
+      try {
+        const results = await this.$axios.$get(`${process.env.API_URL}/report/result/?new_format${dateParam}${zipCodeParam}${countyParam}`)
```

Comment:
lets use query-string module https://www.npmjs.com/package/query-string to generate filter params, so we could have build url out of 

```
const queryParams = {
   date: ...,
   county: ...,
   zipCode: ...
}

queryString.stringify(queryParams)
```

Reply:
ok, gute idee

Reply:
done